.github/workflows/ci.yml

```
@@ -87,6 +87,28 @@ jobs:
             - name: Run the unit tests
               run: vendor/bin/phpunit --colors=always
 
+    php-8:
+        name: PHP 8
```

```suggestion
    php80:
        name: PHP 8.0
```

Maybe we need a `php81` job in the future.